### DISCUSSION TITLE
Style option in dropdown?

### DISCUSSION THREAD

**Discussion by QuangLoc0311** [csandman/chakra-react-select#182] [Q&A] [answered]
How can I change the style of background? 
I also use custom component for options but it's still have default style like this.
And how I can remove "small padding top" in the "first option" and "padding bottom" at "last option" in dropdown

![Screen Shot 2022-09-14 at 10 32 31](https://user-images.githubusercontent.com/32577018/190054041-f201e17b-c5e0-4154-983a-68e561ccf0a4.png)

**Comment by csandman** [comment, marked as answer]
I can't tell exactly what you're trying to do, but what you're probably looking for is the [`chakraStyles`](https://github.com/csandman/chakra-react-select#chakrastyles) prop.  Something along these lines should do the trick, but I'm not sure how you have your custom components set up.

```tsx
return (
  <Select
    chakraStyles={{
      option: (provided) => ({
        ...provided,
        _first: {
          pt: 0
        },
        _last: {
          pb: 0
        }
      })
    }}
  />
)
```

Here's a demo of the prop in action, obviously without the custom components: https://codesandbox.io/s/chakra-react-select-option-styles-6g16yc?file=/example.js

If you need further help, please make a CodeSandbox reproduction of what you have so far and I can give a better example based on that: https://codesandbox.io/s/chakra-react-select-vsvr0?file=/example.js
